src/features/stats/reducer.js

```diff
@@ -19,6 +23,7 @@ const initialState = {
     gold: 0,
     equippedItems: {},
     levelUp: { level: 0, hp: 0, dmg: 0 },
+    start: true,
```

Comment:
Could `start` have a more descriptive name? I'm not entirely sure what its purpose is.

Reply:
Upon further play testing, the later addition of the `moreMana` variable has made the `start` variable redundant and has therefore been removed.